tests/test_examples.py

```diff
@@ -310,11 +315,15 @@ def test(self):
                 env_variables["DEEPSPEED_HPU_ZERO3_SYNC_MARK_STEP_REQUIRED"] = "1"
                 env_variables["PT_HPU_MAX_COMPOUND_OP_SYNC"] = "1"
                 env_variables["PT_HPU_MAX_COMPOUND_OP_SIZE"] = "1"
+            elif fsdp:
+                env_variables["LOWER_LIST"] = str(example_script.parent / "ops_bf16.txt")
```

Comment:
Not every FSDP model will specify this environment variable. Can we change this check to populate this env variable only if "ops_bf16.txt" file is present in example_script.parent directory?

Reply:
We can also use the variable `model_name` that is available in this scope to check the model, see the `elif` right above

Reply:
Change pushed to update the env on model basis.

Reply:
@vivekgoe Does the new change look good to you? If yes, I'll merge this PR